cloud/passportauth/example/index.ts

```diff
@@ -1,13 +1,14 @@
-import {BunyanLogger, Logger} from '@raincatcher/logger';
+import { BunyanLogger, Logger } from '@raincatcher/logger';
```

Comment:
yes, final checking via tslint in CI is the best we can have on top of editor support. How did you manage to configure vscode to add these? I couldn't find a way to